x/evidence/keeper/keeper.go

```diff
@@ -201,13 +201,13 @@ func (k Keeper) MarshalEvidenceJSON(evidence exported.Evidence) ([]byte, error)
 		return nil, err
 	}
 
-	return k.cdc.MarshalJSON(any)
+	return k.cdc.MarshalBinaryBare(any)
```

Comment:
MarshalEvidenceJSON should still do JSON marshaling. But maybe it's not even needed?

Reply:
Ah, I think yes. We can remove this totally.